### ISSUE TITLE
Documentation clarification: status of iOS part of project

### ISSUE THREAD

**Issue by mwpb** [jepiqueau/capacitor-data-storage-sqlite#43]
Firstly, thanks for creating this plugin. This is not so much a bug report but more a request for clarification in the documentation. In both 

https://github.com/jepiqueau/capacitor-data-storage-sqlite/blob/master/docs/API.md

and 

https://github.com/jepiqueau/capacitor-data-storage-sqlite/blob/master/docs/USAGE.md

there is the statement

`iOS for testing only`

Now I have done a very limited test on iOS and based on usage (I'm not able at the moment to directly check the file system for the db file) things appear to be working fine. Is there some reason why the iOS part (or indeed any other part) might not be ready for production use?

Thanks,

**Comment by jepiqueau** [comment]
@mwpb thanks for using the plug-in the iOS part is fully working. Thanks for pointing me this error in the doc. I forgot to remove it when releasing verso 3.0. As I am in vacation can you make a PR I will approve it Many thanks

**Comment by mwpb** [comment]
Hello @jepiqueau thanks I'm going to create a pull request momentarily. This is the first time I've done this so please let me know if there is anything wrong and I'll fix it.

Also, there is no rush from my perspective. Enjoy your vacation!

For the future, I'd be willing to add a section in the docs for vanilla TypeScript (in parallel to Angular that already exists) if that would be useful.

Thanks,

**Comment by jepiqueau** [comment]
@mwpb i merged the PR. Fill free to add a section in the docs for vanilla Typescript. Make a new PR that i will merge

**Comment by mwpb** [comment]
Ok sounds good @jepiqueau . Might take a little while to get around to this but when it is ready I'll put in a PR. As for the current issue, feel free to close as you see fit. Thanks,

**Comment by jepiqueau** [comment]
@mwpb thanks for your contribution I close the issue
